cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/KafkaRoller.java

```diff
@@ -420,35 +427,61 @@ private RestartPlan restartPlan(int podId, Pod pod) throws ForceableProblem, Int
     }
 
     /**
-     * Returns a Future which completes with the config of the given broker.
+     * Returns a config of the given broker.
      * @param ac The admin client
      * @param brokerId The id of the broker.
      * @return a Future which completes with the config of the given broker.
      */
     protected Config brokerConfig(Admin ac, int brokerId) throws ForceableProblem, InterruptedException {
-        ConfigResource resource = new ConfigResource(ConfigResource.Type.BROKER, String.valueOf(brokerId));
+        ConfigResource resource = Util.getBrokersConfig(brokerId);
         return await(Util.kafkaFutureToVertxFuture(vertx, ac.describeConfigs(singletonList(resource)).values().get(resource)),
             30, TimeUnit.SECONDS,
             error -> new ForceableProblem("Error getting broker config", error)
         );
     }
 
-    protected void dynamicUpdateBrokerConfig(int podId, Admin ac, KafkaBrokerConfigurationDiff configurationDiff)
+    /**
+     * Returns logging of the given broker.
+     * @param ac The admin client
+     * @param brokerId The id of the broker.
+     * @return a Future which completes with the logging of the given broker.
+     */
+    protected Config brokerLogging(Admin ac, int brokerId) throws ForceableProblem, InterruptedException {
+        ConfigResource resource = Util.getBrokersLogging(brokerId);
+        return await(Util.kafkaFutureToVertxFuture(vertx, ac.describeConfigs(singletonList(resource)).values().get(resource)),
+                30, TimeUnit.SECONDS,
+            error -> new ForceableProblem("Error getting broker logging", error)
+        );
+    }
+
+    protected void dynamicUpdateBrokerConfig(int podId, Admin ac, KafkaBrokerConfigurationDiff configurationDiff, KafkaBrokerLoggingConfigurationDiff logDiff)
             throws ForceableProblem, InterruptedException {
-        Map<ConfigResource, Collection<AlterConfigOp>> configDiff = configurationDiff.getConfigDiff();
-        log.debug("{}: Altering broker {} with {}", reconciliation, podId, configDiff);
-        AlterConfigsResult alterConfigResult = ac.incrementalAlterConfigs(configDiff);
+        Map<ConfigResource, Collection<AlterConfigOp>> updatedConfig = new HashMap<>(2);
+        updatedConfig.put(Util.getBrokersConfig(podId), configurationDiff.getConfigDiff());
+        updatedConfig.put(Util.getBrokersLogging(podId), logDiff.getLoggingDiff());
+
+        log.info("{}: Altering broker {} with {}", reconciliation, podId, updatedConfig);
+
+        AlterConfigsResult alterConfigResult = ac.incrementalAlterConfigs(updatedConfig);
         KafkaFuture<Void> brokerConfigFuture = alterConfigResult.values().get(new ConfigResource(ConfigResource.Type.BROKER, Integer.toString(podId)));
+        KafkaFuture<Void> brokerLoggingConfigFuture = alterConfigResult.values().get(new ConfigResource(ConfigResource.Type.BROKER_LOGGER, Integer.toString(podId)));
```

Comment:
You just wrote a `Util` method for `new ConfigResource(ConfigResource.Type.BROKER_LOGGER,` so use it!